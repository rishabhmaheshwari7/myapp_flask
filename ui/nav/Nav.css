.nav-container{  
    font-weight: bold;
    display:flex;
    position: absolute;
    top:1px;
    padding-top:25px;
    margin-left: 350px;;
    flex-direction: row;
    width: 60%; 
    align-items: center;
    justify-content: center;
}
.linker{
    color:white;
    text-decoration: none;
    padding:0.5rem 1rem;
    font-size: 25px;
    
}
.linker:hover{
    background-color:#004890;
    color: white;
    border-radius:4px;
    transition:all 0.2s ease-out;
}