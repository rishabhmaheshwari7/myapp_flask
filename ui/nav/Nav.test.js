import { render, screen } from "@testing-library/react"

import { MemoryRouter } from "react-router-dom"

import Nav from "./Nav.component"


describe('Nav Component', () => {

    it('Should render the Nav text correctly', () => {
        render(<MemoryRouter><Nav /></MemoryRouter>)

        expect(screen.getAllByTestId("img")).toMatchSnapshot()

    })
})