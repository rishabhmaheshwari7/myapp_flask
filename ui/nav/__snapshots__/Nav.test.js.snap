// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Should render the header text correctly 1`] = `
Array [
  <div
    class="item-b"
    data-testid="img"
  >
    <a
      class="linker"
      href="/view/results"
    >
      View Results
    </a>
     
    <br />
    <a
      class="linker"
      href="/admin/add/questions"
    >
      Add Questions
    </a>
     
    <br />
    <a
      class="linker"
      href="/delete"
    >
      Delete Questions
    </a>
     
    <br />
  </div>,
]
`;

exports[`Nav Component Should render the Nav text correctly 1`] = `
Array [
  <div
    class="item-b"
    data-testid="img"
  >
    <a
      class="linker"
      href="/view/results"
    >
      View Results
    </a>
     
    <br />
    <a
      class="linker"
      href="/admin/add/questions"
    >
      Add Questions
    </a>
     
    <br />
    <a
      class="linker"
      href="/delete"
    >
      Delete Questions
    </a>
     
    <br />
  </div>,
]
`;
